packages/block-library/src/template-part/variations.js

```diff
@@ -40,6 +40,10 @@ export function enhanceTemplatePartVariations( settings, name ) {
 				'wp_template_part',
 				`${ theme }//${ slug }`
 			);
+
+			if ( entity?.slug ) {
```

Comment:
Should we remove optional chaining `if ( entity.slug )`, I think `if` is enough, how do you think?